src/modules/UI/scenes/CreateWallet/CreateWalletReview.ui.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import React, {Component} from 'react'
+import {Actions} from 'react-native-router-flux'
+import {
+  ActivityIndicator,
+  View,
+  Keyboard
+} from 'react-native'
+import {fixFiatCurrencyCode} from '../../../utils'
+import Text from '../../components/FormattedText'
+import {SecondaryButton, PrimaryButton} from '../../components/Buttons'
+
+import styles, {styles as stylesRaw} from './style.js'
+import s from '../../../../locales/strings.js'
+import Gradient from '../../components/Gradient/Gradient.ui'
+import { createCurrencyWallet } from '../../../../actions/indexActions'
+import type { GuiWalletType, GuiFiatType } from '../../../../types'
+
+const DONE_TEXT = s.strings.fragment_create_wallet_create_wallet
+const BACK_TEXT = s.strings.title_back
+
+export type Props = {
```

Comment:
This should be of format

`export type MyComponentProps = MyComponentOwnProps & MyComponentStateProps & MyComponentDispatchProps`